src/views/EditSimple.vue

```diff
@@ -31,15 +31,23 @@
 		<PopoverLoadingIndicator
 			v-if="isLoading" />
 
-		<Actions
-			v-if="!isLoading"
-			class="event-popover__close-action">
-			<ActionButton
-				icon="icon-close"
-				@click="cancel">
-				{{ $t('calendar', 'Close') }}
-			</ActionButton>
-		</Actions>
+		<div class="event-popover__top-right-actions">
+			<Actions v-if="!isLoading">
```

Comment:
Yes, because i want two separate icons.
If you put multiple Actions into one `Actions` component, it will automatically turn it into a three-dots menu with a drop-down.

Reply:
But what i should do is, put the v-if=isReadOnly on the actions, not the ActionButton.

Reply:
done